include/quda.h

```diff
@@ -623,6 +623,12 @@ extern "C" {
     /** Whether to run null vector oblique checks once set up is complete */
     QudaBoolean run_oblique_proj_check;
 
+    /** Whether to use eigenvectors for the nullspace */
+    QudaBoolean use_low_modes;
+
+    /** Deflate the coarsest grid  */
+    QudaBoolean deflate_coarsest;
```

Comment:
`use_low_modes` and `deflate_coarsest` don't look specific to staggered mg.  Why are these in this branch but not in develop?

Reply:
These are entirely vestigial. No longer in develop because we now use `--mg-eig <level> <true/false>`

Reply:
Good catch. It looks like at some point the copy of `multigrid_evolve_test` in this branch fell (far) out of sync with what's in `develop`, which is why this never threw a warning. I've both removed these declarations from `include/quda.h` and fixed `multigrid_evolve_test`.